packages/tables/src/styled/StyledRow.ts

```diff
@@ -23,36 +23,79 @@ export interface IStyledRowProps {
   size?: ITableProps['size'];
 }
 
+const baseRowColorStyles = ({ theme, isStriped }: IStyledRowProps & ThemeProps<DefaultTheme>) => {
+  return css`
+    border-bottom: ${theme.borders.sm} ${getColor({ variable: 'border.subtle', theme })};
```

Comment:
[nit] the preference is usually to separate `border-bottom` from `border-bottom-color`. And moving these values out to local variables makes the CSS more readable.